docs/own-instance.rst

```diff
@@ -80,7 +80,7 @@ Always use an instance which is operated by people you trust.  The privacy
 features of SearXNG are available to users no matter what kind of instance they
 use.
 
-If someone is on the go or just wants to try SearXNG for the first time public
-instances are the best choices.  Additionally, public instance are making a
-world a better place, because those who cannot or do not want to run an
-instance, have access to a privacy respecting search service.
+For those on the go, or just wanting to try SearXNG for the first time, public
+instances are the best choice.  Additionally, public instance are making the
+world a better place by giving those who cannot, or do not, want to run an
```

Comment:
> "those who cannot, or do not, want to run"

I can't judge, but is this punctuation really correct in English? Why not..

> "those who can't or don't want to run"

Reply:
Hi, sorry, was a bit busy. I'm not a linguist but I'm pretty sure both are correct/acceptable :)

Reply:
I'd keep it with the commas though, as I think it's a subjunctive clause and it flows better.